packages/manager/modules/pci/src/projects/project/storages/databases/translations/Messages_fr_FR.json

```diff
@@ -29,5 +29,7 @@
   "pci_database_node_type_memory": "{{memorySize}} RAM",
   "pci_database_node_type_storage": "{{storageSize}} de stockage SSD",
   "pci_database_node_type_storage_range": "de {{storageMinSize}} à {{storageMaxSize}} de stockage SSD",
-  "pci_database_node_type_label": "Modèle de nœud"
+  "pci_database_node_type_label": "Modèle de nœud",
+  "pci_database_footer_part_1": "Tous les noms de produits, logos et marques sont la propriété de leurs propriétaires respectifs. Tous les noms d'entreprise, de produit et de service utilisés dans ce site Web sont à des fins d'identification seulement. L'utilisation de ces noms, logos et marques n'implique aucune approbation.",
+  "pci_database_footer_part_2": "*Redis est une marque de Redis Ltd. Tous les droits qui y sont attachés sont réservés à Redis Ltd. Toute utilisation par OVH SAS est uniquement à titre de référence et n'indique aucun sponsoring, soutien ou affiliation entre Redis et OVH SAS."
```

Comment:
i would put the asterisk directly in the html template, outside of the translation

Reply:
The translations are already done.

Reply:
@ghyenne Following this request, we can create additional tasks to handle Pull Request comments (via a separate batch).